projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -43,11 +57,22 @@ export class TuiLineClampComponent implements AfterViewInit {
     @tuiDefaultProp()
     content: PolymorpheusContent = '';
 
+    @Input()
+    @tuiDefaultProp()
+    showHint = this.options.showHint;
+
+    @Output()
+    readonly overflowChange: Observable<{isOverflown: boolean}> = this.isOverflown$.pipe(
+        distinctUntilChanged(),
+        map(isOverflown => ({isOverflown})),
```

Comment:
why do we wrap output with extra object ?

May be just
```ts
readonly overflowChange: Observable<boolean>
```

Reply:
It seemed to me that it would be clearer to the external user what exactly comes in the event

Reply:
```html
(overflowChange)="doSomething($event)"
```
It is self-descriptive enough.


But let rename it `overflowChange` => `overflownChange`

Reply:
done